### ISSUE TITLE
Really poor sound quality even when set to "High" Audio quality

### ISSUE THREAD

**Issue by ThornedRose** [pithos/pithos#434]
It seems that after an update to one of Pithos' dependencies (lib-gst I think), the audio quality has become *really* bad. It sounds like it's coming from a slightly mistuned radio and there is massive bass distortion (even though my sub is turned right down).
This only occurs with Pithos and never used to be an issue (about a week ago).

I have also tried Pianobar and it sounds fine (as does local playback through Deepin Music and Clementine).

**Comment by ThornedRose** [comment]
I ran into a problem when installing Pianobar
`ERROR: Unable to open ALSA device 'default' for playback => Invalid argument`
Which was solved by editing /etc/libao.conf to: `default_driver=pulse`
[Original]
`default_driver=alsa
dev=default`
as per https://bbs.archlinux.org/viewtopic.php?pid=1235200#p1235200

I tried Pithos again and it now sounds fine. No idea why only recently/after updates that's suddenly become an issue. But at any rate, following the above fixed the problem. :)
